src/components/Home/HeroSection.tsx

```diff
@@ -36,17 +36,16 @@ const HeroSection = () => {
           Ready to tailor your web experience? Let&apos;s start crafting!
         </h4>
         <div className="mt-8 flex flex-col gap-4 sm:flex-row">
-          <button
-            className="z-10 min-w-[12rem] rounded-md bg-blue-500 px-4 py-2"
-            // onClick={() =>
-            //   alert(`${window.screen.width} ${window.screen.height}`)
-            // }
-          >
-            Explore Components
-          </button>
-          <button className="z-10 min-w-[12rem] rounded-md border px-4 py-2">
-            Get Started
-          </button>
+          <a href="/components" target='_blank'>
```

Comment:
Why use `anchor` tag?
Use `Link` component instead
Also we don't want to open the pages in new tab

Reply:
Ok sir, I will make the necessary changes

Reply:
I have made the necessary changes sir
1. Changed `<a>` tags to `Link` tags
2. Made sure that the pages do not open in a new tab